Product/Production/Adapters/General/CONNECTDirectConfig/src/main/java/gov/hhs/fha/nhinc/directconfig/service/ConfigurationFault.java

```diff
@@ -59,7 +59,7 @@
 
     private String message = "";
 
-    private ConfigurationError error = ConfigurationError.Unknown;
+    private transient ConfigurationError error = ConfigurationError.Unknown;
```

Comment:
Can't find the SONAR defect that mentioned why that variable should have been made transient.  Will update when I find it/remember.